src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -182,6 +189,30 @@ function WorkspaceMembersPage({
         getWorkspaceMembers();
     }, [isOffline, prevIsOffline, getWorkspaceMembers]);
 
+    useEffect(() => {
+        if (isEmptyObject(policyOwnershipChangeChecks?.[policyID])) {
+            return;
+        }
+
+        Policy.requestWorkspaceOwnerChange(policyID);
+    }, [policyID, policyOwnershipChangeChecks]);
+
+    useEffect(() => {
```

Comment:
we can't decide it in the callback assigned to the button, because we don't know this at a time - we get this information asynchronously from the API, so it needs to be done in the `useEffect` that reacts to the change of the `policy.errorFields.changeOwner` property